BackEndFlask/Functions/test_files/test_genericImport.py

```diff
@@ -0,0 +1,803 @@
+            assert user is not None, errorMessage
+            errorMessage = "upload failed to assign lms_id attribute to TA when expected."
+            assert user.lms_id is not None, errorMessage
+            user = get_user_by_email(
+                "teststudent1@gmail.com"
+            )
+            errorMessage = "get_user_by_email() encountered an unexpected error!"
+            assert type(user) is not type(""), errorMessage
+            errorMessage = "genericcsv_to_db() did not correctly create the valid test student"
+            assert user is not None, errorMessage
+            errorMessage = "upload failed to assign lms_id attribute to student when expected."
+            assert user.lms_id is not None, errorMessage                        
+            user_id = get_user_user_id_by_email(
+                "testTA1@gmail.com"
+            )
+            errorMessage = "get_user_user_id_by_email() encountered an unexpected error!"
+            assert type(user_id) is not type(""), errorMessage
+            user_courses = get_user_courses_by_user_id(
+                user_id
+            )
+            errorMessage = "get_user_courses_by_user_id() encountered an unexpected error!"
+            assert type (user_courses) is not type(""), errorMessage
+            errorMessage = "genericcsv_to_db() did not correctly enroll the valid test TA in the test course"
+            assert user_courses.__len__() == 1, errorMessage
+            user_id = get_user_user_id_by_email(
+                "teststudent1@gmail.com"
+            )
+            errorMessage = "get_user_user_id_by_email() encountered an unexpected error!"
+            assert type(user_id) is not type(""), errorMessage
+            user_courses = get_user_courses_by_user_id(
+                user_id
+            )
+            errorMessage = "get_user_courses_by_user_id() encountered an unexpected error!"
+            assert type (user_courses) is not type(""), errorMessage
+            errorMessage = "genericcsv_to_db() did not correctly enroll the valid test student in the test course"
+            assert user_courses.__len__() == 1, errorMessage
+            errorMessage = "deleteAllUsersUserCourses() encountered an unexpected error!"
+            assert type(deleteAllUsersUserCourses(result["course_id"])) is not type(""), errorMessage
+            errorMessage = "deleteOneAdminCourse() encountered an unexpected error!"
+            assert type(deleteOneAdminCourse(result)) is not type(""), errorMessage
+        except:
+            errorMessage = "deleteAllUsersUserCourses() encountered an unexpected error!"
+            assert type(deleteAllUsersUserCourses(result["course_id"])) is not type(""), errorMessage
+            errorMessage = "deleteOneAdminCourse() encountered an unexpected error!"
+            assert type(deleteOneAdminCourse(result)) is not type(""), errorMessage
+            raise
+
+# test_file_not_found()
```

Comment:
Why are there lots of commented out tests? Are they broken or deprecated? Can they be deleted?